nbs/flags_page.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>Should be "use dots for submodules" and also provide an example.</p><p> </p><p>Also do not include the .py extension should be "do not include the .py file extension"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/418/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>